modules/claims_api/app/controllers/concerns/claims_api/document_validations.rb

```diff
@@ -32,10 +32,12 @@ def document_content_type_errors
         end
       end
 
+      # rubocop:disable Layout/LineLength
```

Comment:
can we not just bump the second clause down to the next line to get around the LineLength cop?

Reply:
good point, I'll make that update before merging